radicle-node/src/service.rs

```diff
@@ -359,9 +359,18 @@ where
         for (id, addr) in addrs {
             self.connect(id, addr);
         }
-        // Ensure that our inventory is recorded in our routing table.
-        for id in self.storage.inventory()? {
-            self.routing.insert(id, self.node_id(), time.as_millis())?;
+        // Ensure that our inventory is recorded in our routing table, and we are tracking
+        // all of it. It can happen that inventory is not properly tracked if for eg. the
+        // user creates a new repository while the node is stopped.
```

Comment:
'tracked if for eg.' -> 'tracked eg.'?

Reply:
Hmm that reads differently, it's only true *if* the user create a new repo while the node is stopped..

Reply:
Ok true, its more about 'eg.' being a synonym (?) but not direct replacement for 'example'.  So 'tracked if eg.' or 'tracked if for example'.